SimplenoteUITests/XCTest+Extensions.swift

```diff
@@ -19,3 +19,17 @@ extension XCUIElement {
         self.typeText(text)
     }
 }
+
+extension XCUIElementQuery: Sequence {
```

Comment:
Nitpick. Since this is a port of open source code, let's give credit in the source, too.

This is also useful because, if something breaks here, we can check if the source has been updated or got there to suggest updates.

```suggestion
// Credits to https://github.com/onmyway133/blog/issues/628
extension XCUIElementQuery: Sequence {
```

Reply:
Of course, fully agree. Added. 🙇